src/Test/Perf/util/TraceManager_util.csx

```diff
@@ -0,0 +1,134 @@
+        string scenarioPath = @"%SYSTEMDRIVE%\CPC")
+    {
+        _cpcFullPath = Path.Combine(Environment.ExpandEnvironmentVariables(cpcFolderPath), "CPC.exe");
+        _scenarioGenerator = new ScenarioGenerator(Environment.ExpandEnvironmentVariables(scenarioPath));
+    }
+
+    public static void Run(string testExecutable, string argument)
+    {
+        var processResult = RunProcess(testExecutable, argument);
+        if (processResult.Failed)
+        {
+            throw new SystemException($@"The process {testExecutable} failed. {processResult.StdErr}");
+        }
+    }
+
+    public void Setup()
+    {
+         var processResult = RunProcess(_cpcFullPath, "/Setup /DisableArchive");
+        if (processResult.Failed)
+        {
+            throw new SystemException($@"The process ""CPC.exe /Setup /DisableArchive"" failed. {processResult.StdErr}");
+        }
+    }
+
+    public void Start()
+    {
+        var processResult = RunProcess(_cpcFullPath, "/Start /DisableArchive");
+        if (processResult.Failed)
+        {
+            throw new SystemException($@"The process ""CPC.exe /Start /DisableArchive"" failed. {processResult.StdErr}");
+        }
+    }
+
+    public void Stop()
+    {
+        var processResult = RunProcess(_cpcFullPath, @"/Stop /DisableArchive /ScenarioPath=""%SYSTEMDRIVE%/CPC/scenarios.xml""");
+        if (processResult.Failed)
+        {
+            throw new SystemException($@"The process ""CPC.exe /Stop /DisableArchive"" failed. {processResult.StdErr}");
+        }
+    }
+
+    public void Cleanup()
+    {
+        var processResult = RunProcess(_cpcFullPath, "/Cleanup /DisableArchive");
+        if (processResult.Failed)
+        {
+            throw new SystemException($@"The process ""CPC.exe /Cleanup /DisableArchive"" failed. {processResult.StdErr}");
+        }
+    }
+
+    public void StartScenario(string scenarioName, string processName)
+    {
+        _scenarioGenerator.AddStartScenario(scenarioName, processName);
+    }
+
+    public void StartEvent()
+    {
+        _scenarioGenerator.AddStartEvent(_startEventAbsoluteInstance);
+        _startEventAbsoluteInstance++;
+    }
+
+    public void EndEvent()
+    {
+        _scenarioGenerator.AddEndEvent();
+        _stopEventAbsoluteInstance++;
+    }
+
+    public void EndScenario()
+    {
+        _scenarioGenerator.AddEndScenario();
+    }
+
+    public void EndScenarios()
+    {
+        _scenarioGenerator.AddScenariosFileEnd();
+    }
+
+    public void WriteScenariosFileToDisk()
+    {
+        _scenarioGenerator.WriteToDisk();
+    }
+
+    public void ResetScenarioGenerator()
+    {
+        _scenarioGenerator.Initialize();
+        _startEventAbsoluteInstance = 1;
+        _stopEventAbsoluteInstance = 1;
+    }
+    
+    public void PrintTest()
+    {
+        Console.WriteLine("Test Printing by TraceManager");
+    }
+
+    public static ProcessResult RunProcess(string _cpcFullPath, string args)
```

Comment:
Oops! I will take that back. If I make the helpers static then I should be able to use it!